aws/logs_monitoring/template.yaml

```diff
@@ -142,6 +146,11 @@ Parameters:
     Default: ""
     Description: ARN for the Permissions Boundary Policy
 Conditions:
+  SetDdApiKeySecretFromApiKey:
+    Fn::Not:
+      - Fn::Equals:
+         - Ref: DdApiKey
```

Comment:
Related to my previous comment, probably need to update this to compare DdApiKeySecretArn's value with its default value to determine the condition result.

Reply:
Done!